Yes I know. But this literal is causing a warning with gcc 13:
```
[13/14] Building CXX object EleFitsData/CMakeFiles/EleFitsData_DataUtils_test.dir/tests/src/DataUtils_test.cpp.o
/home/hubert/Work/Space/Euclid/EleFits/EleFitsData/tests/src/DataUtils_test.cpp:168:1: warning: integer constant is so large that it is unsigned
  168 |   BOOST_TEST((offset<std::uint64_t>() == -9223372036854775808));
      | ^ ~~~~~~~~~~~~
```
And the value is strictly the same. The  LONG_LONG_MIN is somehow better defined in the std library. Please note that I have also tried `-9223372036854775808LL` without much success.

--- a/EleFitsData/tests/src/DataUtils_test.cpp
+++ b/EleFitsData/tests/src/DataUtils_test.cpp
@@ -5,8 +5,8 @@
 #include "EleFitsData/DataUtils.h"
 
 #include <boost/test/unit_test.hpp>
-#include <climits>
 #include <cstdint>
+#include <limits>
 
 using namespace Euclid::Fits;
 
@@ -165,7 +165,7 @@ BOOST_AUTO_TEST_CASE(bitpix_bzero_test) {
   BOOST_TEST(bitpix<std::int64_t>() == 64);
   BOOST_TEST(offset<std::int64_t>() == 0);
   BOOST_TEST(bitpix<std::uint64_t>() == 64);
-  BOOST_TEST((offset<std::uint64_t>() == LONG_LONG_MIN));
+  BOOST_TEST((offset<std::uint64_t>() == std::numeric_limits<std::int64_t>::min()));
   BOOST_TEST(bitpix<float>() == -32);
   BOOST_TEST(offset<float>() == 0);
   BOOST_TEST(bitpix<double>() == -64);

--- a/EleFitsData/tests/src/DataUtils_test.cpp
+++ b/EleFitsData/tests/src/DataUtils_test.cpp
@@ -157,11 +157,11 @@ BOOST_AUTO_TEST_CASE(bitpix_bzero_test) {
   BOOST_TEST(bitpix<std::int16_t>() == 16);
   BOOST_TEST(offset<std::int16_t>() == 0);
   BOOST_TEST(bitpix<std::uint16_t>() == 16);
-  BOOST_TEST(offset<std::uint16_t>() == -32768);
+  BOOST_TEST(offset<std::uint16_t>() == std::numeric_limits<std::int16_t>::min());
   BOOST_TEST(bitpix<std::int32_t>() == 32);
   BOOST_TEST(offset<std::int32_t>() == 0);
   BOOST_TEST(bitpix<std::uint32_t>() == 32);
-  BOOST_TEST(offset<std::uint32_t>() == -2147483648);
+  BOOST_TEST(offset<std::uint32_t>() == std::numeric_limits<std::int32_t>::min());
   BOOST_TEST(bitpix<std::int64_t>() == 64);
   BOOST_TEST(offset<std::int64_t>() == 0);
   BOOST_TEST(bitpix<std::uint64_t>() == 64);

